hack/make/.integration-daemon-setup

```diff
@@ -2,4 +2,4 @@
 
 bundle .ensure-emptyfs
 bundle .ensure-frozen-images
-bundle .ensure-httpserver
```

Comment:
Let me spin up my VM with overlay and see what I can figure out

Jess Frazelle wrote:
>
> In hack/make/.integration-daemon-setup
> <https://github.com/estesp/docker/pull/2#discussion_r40240048>:
>
> > @@ -2,4 +2,4 @@
> >
> > bundle .ensure-emptyfs
> > bundle .ensure-frozen-images
> > -bundle .ensure-httpserver
>
> hmm i cant get it to work on overlay :/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/estesp/docker/pull/2/files#r40240048>.